index.js

```diff
@@ -76,6 +82,15 @@ const ContainerService = {
       prefService.set(pref[0], pref[1]);
     });
 
+    if (ss.storage.savedConfiguration.hasOwnProperty("metricsUUID")) {
+      this._metricsUUID = ss.storage.savedConfiguration.metricsUUID;
+    } else {
+      // The add-on was installed before metricsUUID was added, create one
+      this._metricsUUID = uuid.uuid().toString();
+      ss.storage.savedConfiguration["metricsUUID"] = this._metricsUUID;
```

Comment:
We don't need to support old version of this file. Just consider this 'metricsUUID' as part of version 1.